apps/react-workshop/src/Button.stories.tsx

```diff
@@ -124,3 +124,7 @@ Stretch.decorators = [
     </div>
   ),
 ];
+
+export const Loading = () => {
+  return <Button loading>Click me</Button>;
+};
```

Comment:
I think this demo will be better if clicking the button toggles the loading state.

Reply:
Sounds good. It's just that I wanted to test how the transition between the non-loading and loading states looks like and so I had to manually update this story.

Reply:
I added a `Button.loading` example, not used anywhere currently. [`3937d9b`](https://github.com/iTwin/iTwinUI/pull/2058/commits/3937d9b898dd7bfce10fe59b68e983f86d49c9f8)

You can use it in the playground like this:

```jsx
export { ButtonLoadingExample as default } from 'examples';
```